spec/gem_spec.rb

```diff
@@ -103,23 +107,23 @@
       end
     end
 
-    context 'with travis as ci provider' do
+    context 'with github actions as ci provider' do
       before(:all) do
-        described_class.new(gem_name: gem_name, ci_provider: 'travis').create
+        described_class.new(gem_name: gem_name, ci_provider: 'github_actions').create
       end
       after(:all) do
         `rm -rf ./#{gem_name}`
       end
 
-      let(:travis) { File.read("./#{gem_name}/.travis.yml") }
-      let(:expected_travis) { File.read('./lib/rsgem/support/travis.yml') }
+      let(:github_actions) { File.read("./#{gem_name}/.github/workflows/ci.yml") }
+      let(:expected_github_actions) { File.read('./lib/rsgem/support/github_actions.yml') }
 
-      it 'adds travis configuration file' do
-        expect(travis).to eq expected_travis
```

Comment:
shouldn't we keep this test for when the CI provider is travis?

Reply:
Given we actually completely overwrite the file, I decided to always remove it when running this task. 

So that when the default CI provider is set, rsgem will just copy&paste our default file to the correct path `./.travis.yml`

Reply:
why the behavior for github actions is different? I think I'm not completely understanding this, but go ahead

Reply:
It's the same behaviour. 

Keep in mind that these changes only replaced travis by github actions, so we have a specific context for github actions in this test. 

Given travis is the default option, it was already tested in the previous context.